src/com/elixr/training/Main.java

```diff
@@ -58,29 +75,82 @@ public static String readFile(String fileName) {
     }
 
     public static boolean isSupportedFile(File file) {
-        if (file.getName().endsWith(TXTFILE) || file.getName().endsWith(JSONFILE) ) {
+        if (file.getName().endsWith(TXTFILE) || file.getName().endsWith(JSONFILE)) {
             System.out.println("File format supported");
             return true;
         } else {
             System.out.println("File format not supported");
             return false;
         }
     }
-    public static void searchWord(String data, String searchword) {
+
+    public static void searchWord(String inputFilePath, String data, String searchWord) throws SQLException {
```

Comment:
Searching of the word should be done in a thread

Reply:
implemented thread in searchword class